pkg/api/openshiftcluster.go

```diff
@@ -114,6 +114,8 @@ type NetworkProfile struct {
 
 	PodCIDR     string `json:"podCidr,omitempty"`
 	ServiceCIDR string `json:"serviceCidr,omitempty"`
+
+	PrivateIP string `json:"privateIp,omitempty"`
```

Comment:
I think name might be a little bit confusing? As the first thing I think is PrivateIP of the API Server 
```
PrivateIP
PrivateEndpointIP
```

Reply:
agreed; updated, ptal